backend/src/test/java/io/swagger/service/TestFilmService.java

```diff
@@ -75,105 +75,134 @@ private List<Film> createListOfFilms() {
-
-        Film mockFilm2 = new Film();
-        mockFilm2.setId(mockUUID2);
-        mockFilm2.setTitle("Avenger: Finite War");
-        mockFilm2.setLengthInMinutes(210);
-        mockFilm2.setReleaseYear("2022");
-
-        Film mockFilm3 = new Film();
-        mockFilm3.setId(mockUUID3);
-        mockFilm3.setTitle("Apfel: Steve Arbeit");
-        mockFilm3.setLengthInMinutes(95);
-        mockFilm3.setReleaseYear("2015");
 
+        //Assert
         assertEquals(3, createdListMock.size());
-        assertArrayEquals(List.of(mockFilm1, mockFilm2, mockFilm3).toArray(), createdListMock.toArray());
+        assertArrayEquals(List.of(expectedFilm1, expectedFilm2, expectedFilm3).toArray(), createdListMock.toArray());
     }
 
     @Test
     public void testWhenAddAll() {
+        //Arrange
         List<Film> createdListMock = createListOfFilms();
+
+        //Act
         filmServiceMock.addFilms(createdListMock);
 
+        //Assert
         verify(filmRepositoryMock, Mockito.times(1)).saveAll(createdListMock);
         assertEquals(3, mockRepository.size());
     }
 
     @Test
-    public void testWhenGetAllFilms_ThenReturnAll() {
-        List<Film> result = filmServiceMock.getAllFilms();
+    public void testWhenGetAllFilmsIsEmpty_ThenReturnAll() {
+        //Arrange //Act
+        List<Film> filmsFromService = filmServiceMock.getAllFilms();
 
+        //Assert
         verify(filmRepositoryMock, Mockito.times(1)).findAll();
-        assertEquals(0, result.size());
+        assertEquals(0, filmsFromService.size());
+    }
 
+    @Test
+    public void testWhenGetAllFilmsIsNotEmpty_ThenReturnAll() {
+        //Arrange
         List<Film> createdListMock = createListOfFilms();
         filmServiceMock.addFilms(createdListMock);
 
+        //Act
         List<Film> resultWithFilms = filmServiceMock.getAllFilms();
 
-        verify(filmRepositoryMock, Mockito.times(2)).findAll();
+        //Assert
+        verify(filmRepositoryMock, Mockito.times(1)).findAll();
         assertEquals(3, resultWithFilms.size());
         assertArrayEquals(createdListMock.toArray(), resultWithFilms.toArray());
     }
 
     // Change
     @Test
     public void testWhenGetFilmById_ThenFlow() throws NoSuchFileException {
+        //Arrange
         List<Film> createdListMock = createListOfFilms();
         filmServiceMock.addFilms(createdListMock);
 
+        //Act
         Film result = filmServiceMock.getFilmById(mockUUID1);
 
+        //Assert
         verify(filmRepositoryMock, Mockito.times(1)).findById(mockUUID1);
-        assertEquals(createdListMock.get(0), result);
+        assertEquals(createdListMock.getFirst(), result);
     }
 
     // exception name to result
     @Test
     public void testWhenGetFilmByIdButIDNotExist_ThenThrow() {
+        //Arrange //Act
         Exception exception = assertThrows(NoSuchElementException.class,
                 () -> filmServiceMock.getFilmById(UUID.randomUUID()));
         String expected = "Film not found";
```

Comment:
Diese Zeile gehört in den Arrange-Teil. Vielleicht kann man da auch eine Konstante einführen (Konstantennamen werden immer in upper case geschrieben).